src/routes/index.js

```diff
@@ -17,12 +17,10 @@ import { dialog } from '../controllers/dialog.js'
 import { landingPage } from '../controllers/landing.js'
 import { notFoundPage } from '../controllers/notFound.js'
 import { notFound } from '../middleware/error.js'
-import { robotsTxt } from '../middleware/robots.js'
```

Comment:
To reiterate from the PR description:

> Note that this also required removing the robots.txt, since otherwise that would tell search engines not to crawl those headers, apparently.
> 
> See also:
> https://developers.google.com/search/docs/crawling-indexing/block-indexing